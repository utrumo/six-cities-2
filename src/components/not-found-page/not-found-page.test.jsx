import React from 'react';
import renderer from 'react-test-renderer';
import NotFoundPage from './not-found-page.jsx';

it(`Must be rendered correctly after restart`, () => {
  const tree = renderer.create(<NotFoundPage />).toJSON();
  expect(tree).toMatchSnapshot();
});
