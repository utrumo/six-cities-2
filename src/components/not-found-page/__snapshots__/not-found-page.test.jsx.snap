// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Must be rendered correctly after restart 1`] = `
Array [
  <h1>
    404
  </h1>,
  <p>
    Page not found
  </p>,
]
`;
